src/main/java/com/ericsson/ei/notifications/EmailSender.java

```diff
@@ -99,6 +99,11 @@ public MimeMessage prepareEmailMessage(String recipients, String mapNotification
      */
     private MimeMessage prepareEmail(String mapNotificationMessage, String emailSubject,
             String[] recipients) {
+        // Added default sender and subject for email notification
+        if (sender.isEmpty() || subject.isEmpty()) {
+            sender = "noreply@ericsson.com";
```

Comment:
In this case it is not possible because if we make the variable final we have to initiate it directly or in a constructor but we can't use the variables in if block.

Reply:
Ah I meant the value only, for example: private static final String SUBJECT_VALUE = "Email notification";

Reply:
Yah , If that is the case we can use the value but we can't reassign it.
In our Scenario as I am assigning the value after checking the condition so it is not possible to put as static final.

Reply:
yah Emelie I have just tried putting static final and without if condition, its working fine. I will update the change.